cmd/build/v2/build.go

```diff
@@ -199,6 +221,9 @@ func (bc *OktetoBuilder) buildSvcFromDockerfile(ctx context.Context, manifest *m
 		return "", fmt.Errorf("error expanding build args from service '%s': %w", svcName, err)
 	}
 
+	tagToBuild := bc.tagsToCheck(manifest.Name, svcName, buildSvcInfo)[0]
```

Comment:
indexing to the first element of the output of the `tagsToCheck` function makes us too dependent on that function I think. Could we refactor that function and make it more explanatory? When it is used it assumes that certain values will occupy certain positions and it is a bit confusing to read, here and in a few lines ahead

Reply:
agree, Why the first position? If we only use one, why we return a list?